index.html

```diff
@@ -19156,25 +19168,25 @@ <h2 id="AudioWorklet-Sequence">
             <a>AudioWorkletProcessor</a>.
           </p>
           <ol>
-            <li>In the main scope, <code>window.audioWorklet</code> is
-            requested to import a script. No <a>AudioWorkletGlobalScope</a>s
-            exist yet, so the script is fetched and added to the Worklet module
-            responses map.
-            </li>
             <li>An <a>AudioContext</a> is created.
             </li>
-            <li>An <a>AudioWorkletGlobalScope</a> is created in association
-            with the context's audio rendering thread. This is the global scope
-            in which <a>AudioWorkletProcessor</a> class definitions will be
-            evaluated.
+            <li>In the main scope, <code>context.audioWorklet</code> is
+            requested to add a script module.
+            </li>
+            <li>Since none exists yet, a new <a>AudioWorkletGlobalScope</a> is
+            created in association with the context's audio rendering thread.
+            This is the global scope in which <a>AudioWorkletProcessor</a>
+            class definitions will be evaluated.
             </li>
-            <li>As part of the global scope's initialization, the set of
-            imported scripts is run, including the one that was previously
-            imported.
+            <li>The imported script is run in the newly created global scope.
             </li>
             <li>As part of running the imported script, an
             <a>AudioWorkletProcessor</a> is registered under the key
             <code>"Custom1"</code> within the <a>AudioWorkletGlobalScope</a>.
```

Comment:
Didn't notice this before, but explicitly saying "Custom1" is the key here is really confusing because I was reading this to be that the key really is "Custom1".  In fact, I think what you're saying here is that the processor is registered under the user-specified key, **not** actually "Custom1".

This can be addressed in a different PR.

Reply:
Actually +1 on this fix because the diagram had changed to use different variable names (originally, it said Custom1).